WORKSPACE.bazel

```diff
@@ -0,0 +1 @@
+workspace(name = "me_nlohmann_json")
```

Comment:
Other packages are typically using `nlohman_json` or `nlohmann-json`. Please pick one of those over the domain-name-derived package name.

Reply:
I renamed the WORKSPACE to `nloham_json`

Reply:
There's a typo. It should be `nlohmann_json` instead of `nlohman_json`.

Reply:
Whoops. My bad. The `README` needs to be updated as well, while you're at it.